spec/features/mediated_deposit_versioning_spec.rb

```diff
@@ -38,7 +38,9 @@
       click_button('Submit')
 
       visit dashboard_path
-      click_link new_work_title
+      within('.collections') do
```

Comment:
Now that returned/rejected works show in the in progress section of the page, disambiguation is needed here (bc the new title appears twice on the page).